src/step-13-typescript/index.ts

```diff
@@ -0,0 +1,26 @@
+import Fastify, { FastifyInstance, FastifyServerOptions } from 'fastify'
+import loginRoute from './routes/login'
+import usersRoute from './routes/users'
+import authenticatePlugin from './plugins/authenticate'
+
+function buildServer(config: Record<string, unknown>): FastifyInstance {
```

Comment:
let's use the type we created for the configuration here, instead of a generic record